tests/e2e/ctl_v3_grpc_test.go

```diff
@@ -156,16 +156,20 @@ func templateEndpoints(t *testing.T, pattern string, clus *e2e.EtcdProcessCluste
 	return endpoints
 }
 
-func assertAuthority(t *testing.T, expectAurhority string, clus *e2e.EtcdProcessCluster) {
-	var logs []e2e.LogsExpect
-	for _, proc := range clus.Procs {
-		logs = append(logs, proc.Logs())
+func assertAuthority(t *testing.T, expectAuthorityPattern string, clus *e2e.EtcdProcessCluster) {
+	for i := range clus.Procs {
+		line := firstMatch(t, `http2: decoded hpack field header field ":authority"`, clus.Procs[i].Logs())
```

Comment:
I think we can just use `clus.Procs[i].ExpectWithContext` here because in your patch, it doesn't merge member logs.

Reply:
Yeah, it can be simplified.